macros/edr/materializations/test/test.sql

```diff
@@ -1,58 +1,17 @@
-{% macro create_test_result_temp_table() %}
```

Comment:
I just re-ordered the macros in this file and removed one unused one - no logic change.